docs/configure-archive-logs.md

```diff
@@ -59,3 +60,14 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Why doesn't Argo Workflows recommend using this feature?
+
+Argo Workflows log storage facilities are quite basic. It is recommended that you use a combination of:
+
+* A fully featured Kubernetes capable logging facility which will provide you with facilities for indexing, searching and managing of log storage.
```

Comment:
Since this is now stated directly at the top and this is now a "Suggested Alternatives" section, we can remove the first sentence entirely.

We can combine the first bullet as well, since we're just directly stating this now:

```suggestion
We recommend using a Kubernetes-aware utilities which are optimized for indexing, searching, and storing logs, such as [`fluentd`](https://github.com/fluent/fluentd), [Splunk](https://splunkbase.splunk.com/app/4497), [ELK](https://www.elastic.co/elastic-stack/), and others.

```
- use active voice, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice)
- prefer 1 sentence per line, per [style guide](https://argo-workflows.readthedocs.io/en/release-3.5/doc-changes/)
- "facility which will provide you with facilities" is redundant
- more simple and direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language).
- provide some examples -- feel free to expand or change those

Reply:
I've left in the first sentence for people who are linked directly to this from external sources, or who don't read the entire page.
I've dropped splunk and popped in the grafana stack as being open source.
Otherwise I hope I've captures the spirit of what you're after here.

Reply:
See follow-up [below](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1483684845)